tweets/2021-04/yt-espresso.tweet

```diff
@@ -0,0 +1 @@
+Operate First beings users, developers and operators closer together! Check out https://www.youtube.com/playlist?list=PL8VBRDTElCWpneB4dBu4u1kHElZVWfAwW, our new YouTube Espresso series to learn what Operate First is all about and learn about how we build software and perform data science in the open through short videos.
```

Comment:
Operate First brings users, developers and operators closer together! Check out our new YouTube Espresso series to learn about Operate First and how we perform data science in a cloud-native way. https://www.youtube.com/playlist?list=PL8VBRDTElCWpneB4dBu4u1kHElZVWfAwW

Reply:
@durandom changes made 🙂